recipes/libe57format/all/patches/fix-cmake-package-files.patch

```diff
@@ -0,0 +1,19 @@
+--- a/CMakeLists.txt
```

Comment:
Remove this patch. Remove the installed cmake files directly.

Reply:
Fixed